### PR TITLE
Animation Silhouettes

### PR DESCRIPTION
**Pull request by tealefristoe**
This PR adds silhouettes to animations.
- There are five silhouette types and one is chosen randomly on load.
- All videos are disabled until the user clicks the button in the silhouette.
- Pushing the button will zoom in on the organ and then enable videos related to it.

### CODE REVIEW

File: src/components/animation-app.tsx [hunk 1]
```diff
@@ -31,6 +33,7 @@ export const AnimationApp = ({ ac, setKeyVisible }: AnimationAppProps) => {
     console.log(`${ac.o("ANICONTROL2LABEL")}: ${ac.o(control2 ? "ANICONTROL2OPTION2" : "ANICONTROL2OPTION1")}`);
```
Comment (bacalj) [inline src/components/animation-app.tsx]: I'm guessing you'll keep using these `console.log`s for a bit?

File: src/components/silhouette-pane.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    }
+  };
+
+  const instructionsMessage = ac.t("SILHOUETTEINSTRUCTIONS").replace("ORGAN", ac.organ);
+  const title = ac.t("SILHOUETTETITLE").replace("ORGAN", ac.organ[0].toUpperCase() + ac.organ.slice(1));
+  return (
+    <div className="silhouette-pane">
+      {sd && <img src={sd.image} className="silhouette-profile" style={silhouetteStyle} />}
+      <button className={clsx("silhouette-button", ac.organ)} onClick={handleClick} style={buttonStyle} />
+      {sod && ac.organ !== "nose" &&
```
Comment (bacalj) [inline src/components/silhouette-pane.tsx]: Maybe add a comment about why the exception for the nose for future changes?

File: src/components/silhouette-pane.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+import { clsx } from "clsx";
+
+import { PaneTitle } from "./pane-title";
+import { AppContext } from "../hooks/use-app-context";
+import { ISilhouetteData, ISilhouetteOrganData, silhouetteData, silhouetteOrganData }
+  from "../assets/app-data/silhouette-data";
+
+import "./silhouette-pane.scss";
+
+// Make sure you update $transition-duration in silhouette-pane.scss if you change this.
```
Comment (bacalj) [inline src/components/silhouette-pane.tsx:11]: This is clear and commented in both places so I think it makes sense for maintainability.

**Review by bacalj — APPROVED** [review]
👍 Super clean and clear and works.  My questions are small ones and not blockers.

File: src/components/silhouette-pane.scss [hunk 4]
```diff
@@ -0,0 +1,51 @@
+@use './vars';
```
Comment (kswenson) [inline src/components/silhouette-pane.scss:1]: Nice -- it looks like we should be replacing our `@import`s with `@use`s across the board.

File: src/components/silhouette-pane.tsx [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+export const SilhouettePane = ({ ac, hasZoomed, setHasZoomed }: ISilhouettePane) => {
+  const [silhouetteStyle, setSilhouetteStyle] = useState<Record<string, any>>({});
+  const [buttonStyle, setButtonStyle] = useState<Record<string, any>>({});
+  const [organStyle, setOrganStyle] = useState<Record<string, any>>({});
+  const [zooming, setZooming] = useState(false);
+
+  // Determine which silhouette to use
+  const sdIndex = useMemo(() => {
+    const index = Math.floor(Math.random() * silhouetteData.length);
+    return index < silhouetteData.length ? index : 0;
```
Comment (kswenson) [inline src/components/silhouette-pane.tsx:28]:
```suggestion
    return Math.floor(Math.random() * silhouetteData.length);
```
Shouldn't need the additional test as `Math.random()` returns value in [0,1].

File: src/components/silhouette-pane.scss [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+// Make sure to update zoomDuration in silhouette-pane.tsx if you change this.
```
Comment (kswenson) [inline src/components/silhouette-pane.scss:3]: FYI: It is possible to import constants from .scss to JavaScript with a bit of configuration, but the comments are fine as well.

**Review by kswenson — APPROVED** [review]
👍 Looks good!

**Pull request merged**